### PR TITLE
docs: Fix tables Markdown format

### PR DESCRIPTION
**Pull request by tuxinaut**
#### Description

This PR fixes wrong formatted markdown tables!.

#### Motivation and Context

Markdown tables are written as supposed to be.

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [x] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

**Comment by vladimyr** [comment]
I'm sorry for asking such a trivial question but it is hard to get the picture of what exactly you did while scrolling through on mobile. It seems that you aligned the table gridlines everywhere, right?

**Comment by tuxinaut** [comment]
@vladimyr don't be sorry it's a good question.

> It seems that you aligned the table gridlines everywhere, right?

Yes, that is absolutely correct.

File: docs/config/README.md [hunk 1]
```diff
@@ -994,7 +994,7 @@ This is based on the `~/.config/gcloud/active_config` file and the `~/.config/gc
 | Option           | Default                                        | Description                                                     |
 | ---------------- | ---------------------------------------------- | --------------------------------------------------------------- |
 | `format`         | `'on [$symbol$account(\($region\))]($style) '` | The format for the module.                                      |
-| `symbol`         | `"☁️ "`                                        | The symbol used before displaying the current GCP profile.      |
+| `symbol`         | `"☁️ "`                                         | The symbol used before displaying the current GCP profile.      |
```
Comment (chipbuster) [inline docs/config/README.md:997]: Strangely, neither the old nor the new alignment seems to be correct here. Is this an artifact of how things render on web, or is the cloud emoji one of them weird double-width emojis that `wcswidth` miscalculates?

**Review by chipbuster — APPROVED** [review]
I'm not opposed to merging this in, but I'd like to note that there seems to be some sort of disagreement between editors used by our contributors as to exactly where these gridlines should be. I feel like I've seen a *lot* of PRs where in addition to making whatever actual changes were needed in the docs, the gridlines also got randomly shifted around a little bit.

I can't seem to find an example of this right now, but it would be lovely to know exactly why this keeps happening, and if there's anything we can do (add a formatter configuration file to the repo somewhere?) to stop it from happening again.

(FWIW most of the issues fixed in this PR seem to be strictly from misalignments, though I'm not sure how some of them could have gotten in there except for a different formatter gone crazy).

File: docs/config/README.md [hunk 2]
```diff
@@ -1657,13 +1657,13 @@ To enable it, set `disabled` to `false` in your configuration file.
...
-| ----------- | ---------------------- | -------------------------------------------------------- |
-| `threshold` | `75`                   | Hide the memory usage unless it exceeds this percentage. |
-| `format`    | `"via $symbol [${ram}( | ${swap})]($style) "`                                     | The format for the module. |
-| `symbol`    | `"🐏"`                 | The symbol used before displaying the memory usage.      |
-| `style`     | `"bold dimmed white"`  | The style for the module.                                |
-| `disabled`  | `true`                 | Disables the `memory_usage` module.                      |
+| Option      | Default                                       | Description                                              |
+| ----------- | --------------------------------------------- | -------------------------------------------------------- |
+| `threshold` | `75`                                          | Hide the memory usage unless it exceeds this percentage. |
+| `format`    | `"via $symbol [${ram}( | ${swap})]($style) "` | The format for the module.                               | $
```
Comment (vladimyr) [inline docs/config/README.md:1663]: Well, the original table was broken but this one is too. It has an extra column. Where did that dollar symbol come from? :confused:

**Review by vladimyr — CHANGES_REQUESTED** [review]

File: docs/config/README.md (hunk 1 above)
Comment (vladimyr) [reply docs/config/README.md:997]: That is just the way it is with emojis, they simply don't play well with monospaced fonts. Glitches like that are kinda expected.

**Comment by vladimyr** [comment]
@tuxinaut What editor have you used for making those changes? I assume you haven't done all that alignment changes manually? :upside_down_face:

**Comment by tuxinaut** [comment]
@vladimyr 

> What editor have you used for making those changes? I assume you haven't done all that alignment changes manually?

Vim search plus [tabular](https://github.com/godlygeek/tabular)

**Comment by davidkna** [comment]
By the way, does anyone know if these changes will undo all translation work for these tables?

**Comment by vladimyr** [comment]
> By the way, does anyone know if these changes will undo all translation work for these tables?

Uh, that is a good and scary question. :thinking: Have no idea :shrug:

**Comment by matchai** [comment]
> By the way, does anyone know if these changes will undo all translation work for these tables?

It does, but that is quickly fixed by running Crowdin's translation memory replacement.
I have to do it every month or so to catch indentation changes like these ones.

**Comment by tuxinaut** [comment]
@vladimyr 

> Well, the original table was broken but this one is too. It has an extra column. Where did that dollar symbol come from? confused

It's now fixed

**Review by andytom — APPROVED** [review]

**Comment by andytom** [comment]
Hey @tuxinaut, this LGTM if you can give this a quick rebase to fix the conflicts we should be good to merge this.

**Comment by tuxinaut** [comment]
@andytom, the rebase was not as quick as expected, but now it's done.

**Comment by andytom** [comment]
Thank you for fixing this @tuxinaut

**Comment by tuxinaut** [comment]
@andytom every time again :smile:

**Pull request merged**